client/src/interfaces.ts

```diff
@@ -85,10 +87,15 @@ export interface GMapPlace {
   geometry: {
     // Encodes location information for map display
     location: {
-      lat: number;
-      lng: number;
+      lat: () => number; // Note: These are functions!
+      lng: () => number; // this is not entirely clear from the Places API documentation
```

Comment:
could you explain a bit more here

Reply:
#116 Figured out I was looking into the wrong place for documentation :sweat_smile: Will amend this comment